resources/language/resource.language.de_de/strings.po

```diff
@@ -1026,21 +1026,21 @@ msgstr "Aktiviere Jellyfin für Kodi"
 
 msgctxt "#33200"
 msgid "Select the libraries to remove"
-msgstr "Markieren Sie die zu löschenden Bibliotheken"
+msgstr "Markiere die zu löschenden Bibliotheken"
 
 msgctxt "#33199"
 msgid "Select the libraries to repair"
-msgstr "Markieren Sie die zu reparierenden Bibliotheken"
+msgstr "Markiere die zu reparierenden Bibliotheken"
 
 msgctxt "#33198"
 msgid "Select the libraries to update"
-msgstr "Markieren Sie die zu aktualisierenden Bibliotheken"
+msgstr "Markiere die zu aktualisierenden Bibliotheken"
 
 msgctxt "#33197"
 msgid "A sync is already running, please wait until it completes and try again."
 msgstr ""
-"Die Synchronisation läuft bereits, warten Sie bitte bis es fertig ist und "
-"versuchen sie es später erneut."
+"Die Synchronisation läuft bereits, warte bitte bis es fertig ist und "
```

Comment:
bis sie

Reply:
Maybe also "Eine Synchronisation" (--> "A sync") instead of "Die Synchronisation" ("The sync")?

I'd replace "es" with "diese"

"Eine Synchronisation läuft bereits, warte bitte bis diese fertig ist und "

Reply:
Even better :)